applications/CompressiblePotentialFlowApplication/python_scripts/compute_lift_process.py

```diff
@@ -46,14 +46,14 @@ def ExecuteFinalizeSolutionStep(self):
         RX = rx/self.reference_area
         RY = ry/self.reference_area
 
-        Cl = RY
-        Cd = RX
+        self.Cl = RY
+        self.Cd = RX
```

Comment:
not sure if applicable, but maybe it is better to make it accessible through a function to avoid unintended access to old values

=> e.g. introduce functions `_CalculateCl` etc